Source/Core/VideoCommon/VertexManagerBase.cpp

```diff
@@ -757,6 +757,16 @@ void VertexManagerBase::OnDraw()
 {
   m_draw_counter++;
 
+  // If the last efb copy was too close to the one before it, don't forget about it until the next
+  // efb copy happens (which might not be for a long time)
+  u32 diff = m_draw_counter - m_last_efb_copy_draw_counter;
+  if (m_unflushed_efb_copy && diff > MINIMUM_DRAW_CALLS_PER_COMMAND_BUFFER_FOR_READBACK)
+  {
+    g_renderer->Flush();
+    m_unflushed_efb_copy = false;
```

Comment:
Do we need a `m_unflushed_efb_copy = false;` on line 770?

Reply:
No, we want to leave it true there, since we haven't flushed yet.  We want this to happen:

|m_unflushed_efb_copy|diff|flush|
|:--:|:--:|:--:|
|true|1||
|true|2||
|...|||
|true|8||
|true|9||
|true|10|!|
|false|1||
|false|2||
|...|||

If you reset it on 770, this would happen:
|m_unflushed_efb_copy|diff|flush|
|:--:|:--:|:--:|
|true|1||
|false|2||
|...|||

Reply:
I must have mistyped  I meant 780.  Where we call `g_renderer->Flush()` so presumably there's no efbs left to flush.

Reply:
Ahh that makes sense